scientist-softserv/harvard-curiosity
# Hide "from this exhibit" option from image source upload

**Hide "from this exhibit" option from image source upload**
* * *

**JIRA Ticket**:
- #248 -- Curation: Image Source

# What does this Pull Request do?
Visually hides the option to select an image source "from this exhibit" from the admin panel because it doesn't work with how we harvest/deliver images currently.

# How should this be tested?
Navigate to the admin panel areas mentioned below and look under the "Image Source" section. There should now only be one option available (a radio button to "upload an image"):
- Appearance > Exhibit Masthead
- Appearance > Exhibit Thumbnail
- Browse > Thumbnail
- Edit mode for a page > Thumbnail tab

Under Appearance --> Visual Theme, switch to the "Vanilla Spotlight" theme and navigate back to any of the page areas mentioned above. The option to select an image "From this exhibit" should be present under the "Image Source" section (this will be useful when we do further testing to get this functionality working).

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? NA
- integration tests? NA

# Interested parties
Tag (@ mention) interested parties
